app/support/DbAdapter.js

```diff
@@ -1,8 +1,12 @@
 import _ from 'lodash'
 import validator from 'validator'
 import NodeCache from 'node-cache'
+import cacheManager from 'cache-manager'
+import redisStore from 'cache-manager-redis'
 import { promisifyAll } from 'bluebird'
-import pgFormat from 'pg-format';
+import pgFormat from 'pg-format'
```

Comment:
we try to slowly migrate to the code-style with semicolons. so it is a good idea to add them (not a requirement, for now), but it's a bad idea to remove them :)